functions/resources/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Their words rearranged</title>
```

Comment:
I love the capitalization here. Like really.

Reply:
Thats a-my-bad for copy pasta-ing.

Reply:
Huh? I was being serial. I like titles starting with a cap and then following words are lower-cased.